benchmark/blstOps.ts

```diff
@@ -279,6 +280,50 @@ const msg = Buffer.from("Mr F was here");
     });
   }
 
+  await runner.run({
```

Comment:
there is a bottleneck in deserialization and sigVerify for attestations.  is there a performance difference for that process?  I roughed out the function and added the perf condition below but need to add a bit more to get it compared to existing SWIG

```
run: (serialized: Uint8Array) => {
    const sig = next.Signature.fromBytes(serialized);
    sig.sigVerify();
}
```

Reply:
I'm getting
```
BLS verification                                                      244.3904 ops/s       4091814 ns/op    489 runs
BLS verification - next                                               449.8385 ops/s       2223020 ns/op    512 runs
```

Reply:
> This benchmark is not an apples-to-apples comparison.

Nope, only built the `next` version and ran out of daylight to infill the SWIG version